run-tests.sh

```diff
@@ -90,17 +90,25 @@ create_scanner_cache() {
 test_scanner() {
   local scanner_finished_successfuly=no container_name
   local image="$1"
+  local args=""
   container_name=$(generate_id)
   info "testing image $1 in container $container_name"
 
   git clone https://github.com/SonarSource/sonar-scanning-examples.git "$container_name"
 
   scanner_props_location="$PWD/$container_name/sonarqube-scanner/sonar-project.properties"
   echo "sonar.projectKey=$container_name-test" >> "$scanner_props_location"
-  echo "sonar.host.url=http://${sonarqube_container_name}:9000" >> "$scanner_props_location"
-  echo "sonar.userHome=/usr/.sonar" >> "$scanner_props_location"
+  if [ "$((RANDOM % 2))" = 1 ]; then 
```

Comment:
wow what is that ?

Reply:
a random boolean :)

Reply:
It seems to me that there are two things to test here in two different test rather than use random.

Reply:
Reading Pierre's mind, I think the question is about the relevancy of random tests.

* the test is complex to read, so hard to maintain
* if the two cases make sense, then two tests should be implemented
* random tests are a bad idea. Errors will be considered as false-positives.

Reply:
I don't follow the stream of thoughts "random test" => "failure considered as FP"

nevertheless, I'm removing test of the case which is neither documented nor advised